typescript/web/src/hooks/use-image-navigation.fixtures.ts

```diff
@@ -0,0 +1,77 @@
+import { GET_ALL_IMAGES_OF_A_DATASET_QUERY } from ".";
+import {
+  GetAllImagesOfADatasetQuery,
+  GetAllImagesOfADatasetQueryVariables,
+  GetAllImagesOfADatasetQuery_dataset_images,
+} from "../graphql-types/GetAllImagesOfADatasetQuery";
+import {
+  ApolloMockResponse,
+  ApolloMockResponses,
+} from "../utils/tests/apollo-mock";
+import { BASIC_DATASET_DATA } from "../utils/tests/data.fixtures";
+import { CURRENT_IMAGE } from "./test-utils";
+
+export const IMAGE_1_DATA = {
+  id: "929f5445-812c-4679-86bd-0f0748679f38",
+  url: "https://labelflow.ai/static/icon-512x512.png",
+  thumbnail200Url: "https://labelflow.ai/static/icon-512x512.png",
+};
+
+export const IMAGE_2_DATA = {
+  id: "51e00f79-d178-4ea1-8bb5-b0fa647f9d19",
+  url: "https://labelflow.ai/static/icon-512x512.png",
+  thumbnail200Url: "https://labelflow.ai/static/icon-512x512.png",
+};
+
+export const IMAGE_3_DATA = {
+  id: "8067a965-d017-4c4b-a82d-98cc803d1020",
+  url: "https://labelflow.ai/static/icon-512x512.png",
+  thumbnail200Url: "https://labelflow.ai/static/icon-512x512.png",
+};
+
+const CURRENT_IMAGE_IS_FIRST = [CURRENT_IMAGE, IMAGE_1_DATA, IMAGE_2_DATA];
+const CURRENT_IMAGE_IS_SECOND = [IMAGE_1_DATA, CURRENT_IMAGE, IMAGE_2_DATA];
+const CURRENT_IMAGE_IS_LAST = [IMAGE_1_DATA, IMAGE_2_DATA, CURRENT_IMAGE];
+const CURRENT_IMAGE_NOT_PRESENT = [IMAGE_1_DATA, IMAGE_2_DATA, IMAGE_3_DATA];
```

Comment:
```suggestion

const EXAMPLE_URL = "https://labelflow.ai/static/icon-512x512.png";

const IMAGES_IDS = [
  "dd5f0305-3d69-457d-9a77-4a46cec0e10b",
  "929f5445-812c-4679-86bd-0f0748679f38",
  "51e00f79-d178-4ea1-8bb5-b0fa647f9d19",
  "8067a965-d017-4c4b-a82d-98cc803d1020",
];

export const [CURRENT_IMAGE_DATA, IMAGE_1_DATA, IMAGE_2_DATA, IMAGE_3_DATA] =
  IMAGES_IDS.map((id) => ({
    id,
    url: EXAMPLE_URL,
    thumbnail200Url: EXAMPLE_URL,
  }));

const CURRENT_IMAGE_IS_FIRST = [CURRENT_IMAGE_DATA, IMAGE_1_DATA, IMAGE_2_DATA];
const CURRENT_IMAGE_IS_SECOND = [IMAGE_1_DATA, CURRENT_IMAGE_DATA, IMAGE_2_DATA];
const CURRENT_IMAGE_IS_LAST = [IMAGE_1_DATA, IMAGE_2_DATA, CURRENT_IMAGE_DATA];
const CURRENT_IMAGE_NOT_PRESENT = [IMAGE_1_DATA, IMAGE_2_DATA, IMAGE_3_DATA];
```

Reply:
Looks a bit much for a test like that

Reply:
It's common factorization... At-least it takes less space (_don't forget CURRENT_IMAGE_ 😉) and don't duplicate the URLs everywhere...